README.md

```diff
@@ -31,6 +31,92 @@ See the [API documentation](http://docs.confluent.io/current/clients/confluent-k
 
 **License**: [Apache License v2.0](http://www.apache.org/licenses/LICENSE-2.0)
 
+
+Examples
+========
+
+High-level balanced consumer
+
+```golang
+import (
+	"fmt"
+	"github.com/confluentinc/confluent-kafka-go/kafka"
+)
+
+func main() {
+
+	c, err := kafka.NewConsumer(&kafka.ConfigMap{
+		"bootstrap.servers": "localhost",
+		"group.id":          "myGroup",
+		"auto.offset.reset": "earliest",
+	})
+
+	if err != nil {
+		panic(err)
+	}
+
+	c.SubscribeTopics([]string{"myTopic", "^aRegex.*[Tt]opic"}, nil)
+
+	for true {
+		msg, err := c.ReadMessage(-1)
```

Comment:
should this be a duration?

Reply:
go coerces it automatically

Reply:
if you'd had set the assigned the time `var i int = -1` you'd have to convert it yourself